src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -2230,18 +2251,14 @@ private boolean copyPC2Sandbox() {
         //point to the PC2 Internal Sandbox file (under "/sandbox" in the home, i.e. installation, directory)
         String srcFileName = home + File.separator + "sandbox" + File.separator + Constants.PC2_INTERNAL_SANDBOX_PROGRAM_NAME ;
         
-        boolean success;
         try {
             //copy the PC2 internal sandbox program into the execute directory
-            success = ExecuteUtilities.copyFile(srcFileName, targetFileName, getLog());
+            ExecuteUtilities.copyFile(srcFileName, targetFileName, getLog());
             
         } catch (Exception e){
             log.severe("Exception copying PC2 Internal Sandbox to execute directory: " + e.getMessage());
-            success = false ;
```

Comment:
Fix this.  This exception should be added to executionData,  as others are, by adding this line:
executionData.setExecutionException(e);

Reply:
Agreed; fixed and pushed.